07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,51 @@
+body {
```

Comment:
It looks like you are some CSS short of completing this assignment. Let me know if you want to get together and work through this one-on-one. We can get it completed and get your grade up to 100.